app.json

```diff
@@ -20,10 +20,15 @@
     "android": {
       "googleServicesFile": "./google-services.json",
       "adaptiveIcon": {
-        "foregroundImage": "./app/assets/images/adaptive-icon.png",
+        "foregroundImage": "./app/assets/images/icon_trans.png",
         "backgroundColor": "#ffffff"
       },
-      "package": "com.strive"
+      "package": "com.strive",
+      "config": {
+        "googleMaps": {
+          "apiKey": "AIzaSyCEhgdrq6tfUEErN326j50qTIeV3uxzmDQ"
```

Comment:
I see we still have API key in our codebase which isn't great could you enlighten me about this ?

Reply:
For sure! This is a very bad habit, however it will need a PR by itself because it is not the only occurrence of it. In our specific context, the data is not sensitive and the Google account is not linked to any payment method.

Reply:
Okay I see I'll allow it this time but behave 👑